frontend/modelarchive/src/main/java/org/pytorch/serve/archive/ModelArchive.java

```diff
@@ -44,7 +45,8 @@ public ModelArchive(Manifest manifest, String url, File modelDir, boolean extrac
         this.extracted = extracted;
     }
 
-    public static ModelArchive downloadModel(String modelStore, String url)
+    public static ModelArchive downloadModel(
+            List<String> urlPatternlist, String modelStore, String url)
```

Comment:
Can we use consistent naming convention i.e. whiteListedURLs

Reply:
Done